File: app/src/main/java/tech/hamid/bookery/MyStory.java
package tech.hamid.bookery;

import java.io.Serializable;

public class MyStory implements Serializable {
    String title;
    String story;

    String color;

    public MyStory() {

    }
    public MyStory(String title, String story, String color) {
        this.title = title;
        this.story = story;
        this.color = color;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getStory() {
        return story;
    }

    public void setStory(String story) {
        this.story = story;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }
}

File: app/src/main/java/tech/hamid/bookerykotlin/MyStory.kt
package tech.hamid.bookerykotlin

import java.io.Serializable

class MyStory : Serializable{
    var title: String? = null
    var story: String? = null
    var color: String? = null

    constructor() {}

    constructor(title: String, story: String, color: String) {
        this.title = title
        this.story = story
        this.color = color
    }
}
